### PR TITLE
make: log updated groups and last package

### LINKED ISSUE #72: make 'make update' log updated packages
When running ./update the newly updated packages are not logged, this might be useful for debugging. Another option is to only log the packages which also have an AVG matched to it.
```
Aug 15 16:13:17 apollo make[14741]: ./db_create
Aug 15 16:13:17 apollo make[14741]: ./update
Aug 15 16:13:24 apollo make[14741]: [+] Update pacman db...
Aug 15 16:13:24 apollo make[14741]: [+] Update package cache...
Aug 15 16:13:24 apollo make[14741]:     --> Querying alpm database...
Aug 15 16:13:24 apollo make[14741]:     --> Collecting pkgbases...
Aug 15 16:13:24 apollo make[14741]:     --> Updating database cache...
Aug 15 16:13:24 apollo make[14741]: [+] Update group status...
```

### PR DESCRIPTION
**Pull request by anthraxx**
Fixes #72

```
[+] Update pacman db...
[+] Update package cache...
    --> Querying alpm database...
    --> Collecting pkgbases...
    --> Latest package: example 0.1.2-1 Wed Oct 11 01:52:41 2017
    --> Updating database cache...
[+] Update group status...
    --> Updated AVG-9: Vulnerable -> Fixed
```

### CODE REVIEW

File: update [hunk 1]
```diff
@@ -61,12 +71,20 @@ def update_package_cache():
     print('    --> Collecting pkgbases...')
     # query all pkgbases per arch
     for arch in archs:
-        cmd = ['expac', '-S', '--config', get_configpath(arch), '%n %e']
+        cmd = ['expac', '-t', '%s', '-S', '--config', get_configpath(arch), '%n %e %v %b']
```
Comment (jelly) [inline update]: Hmm why aren't we using pyalpm for this stuff?

**Comment by anthraxx** [comment]
If I recall there is no way to properly get pkgbase data stuff

**Comment by jelly** [comment]
@anthraxx no problem, I'll add that to my todolist I believe pyalpm can use some improvements :)

**Comment by anthraxx** [comment]
@jelly cool, that would be great

File: update (hunk 1 above)
Comment (jelly) [reply update]: It has been added, so the next release we can start using it. https://github.com/archlinux/pyalpm/commit/f1513b49c3e2af9741fc1a65487b6413a4261de6

**Pull request merged**